lib/plugins/aws/deploy/lib/cleanupS3Bucket.js

```diff
@@ -28,7 +29,7 @@ module.exports = {
 
   async removeObjects(objectsToRemove) {
     if (!objectsToRemove || !objectsToRemove.length) return;
-    this.serverless.cli.log('Removing old service artifacts from S3...');
+    legacy.log('Removing old service artifacts from S3...');
```

Comment:
I think we miss setup of "Updating" phase for main progress, not sure where exactly it should be placed, but I think it's sort of replacement for above log, and afaik currently still `Updating CloudFormation stack` will be presented at this point

Reply:
I added it as a part of ```aws:deploy:finalize:cleanup``` for now but it might be subject to change after we move `display` part, it might also depend on how plugins hook into our logic